.gitlab-ci.yml

```diff
@@ -512,6 +563,38 @@ run_dogstatsd_arm_size_test:
   script:
     - inv -e dogstatsd.size-test --skip-build
 
+.system-probe_build_common:
+  before_script:
+    # Hack to work around the cloning issue with arm runners
+    - mkdir -p $GOPATH/src/github.com/DataDog
+    - '[[ "$ARCH" == arm64 ]] && cp -R $GOPATH/src/github.com/*/*/DataDog/datadog-agent $GOPATH/src/github.com/DataDog'
+    - cd $SRC_PATH
+    - inv -e deps --verbose --dep-vendor-only
+    # Retrieve libbcc from S3
+    - $S3_CP_CMD $S3_ARTIFACTS_URI/libbcc-$ARCH.tar.xz /tmp/libbcc.tar.xz
+    - tar -xvf /tmp/libbcc.tar.xz -C /usr
```

Comment:
This is a bit dirty :/

Reply:
This was to have `inv system-probe.test` able to find `libbcc` like any other system library it is depending on.
Note that this step is not building any delivery. So that has no implication.
Would you prefer to have `libbcc` unarchived in a dedicated directory (`/opt/libbcc` or `/opt/datadog-agent/embedded`) and to pass appropriate `CGO_CFLAGS` `GCO_LDFLAGS` to `inv system-probe.test` ?

Reply:
The cleanest solution I can think of is to also use omnibus to build the system-probe, so omnibus downloads this and puts it into the omnibus build prefix... but it's probably a pain in the ass to implement :P